### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by Arturoo0**

### CODE REVIEW

File: frontend/src/components/Nav/Nav.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+ul li:hover {
+  cursor: pointer;
+}
+
+.navbar-brand {
+  font-size: 3rem;
+}
+
+
```
Comment (Bruception) [inline frontend/src/components/Nav/Nav.css:17]: This spacing :(

File: frontend/src/pages/Test/Test.jsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ export default class Test extends React.Component {
     const { users } = this.state;
     return (
       <div>
+        <Nav links={['test1', 'test2']} />
         <ul>
           { _.map(users, (user, index) => <li key={ index }>{ user.name }</li>) }
```
Comment (Bruception) [inline frontend/src/pages/Test/Test.jsx:29]: Users?

File: frontend/src/pages/Test/Test.jsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ export default class Test extends React.Component {
     const { users } = this.state;
     return (
       <div>
+        <Nav links={['test1', 'test2']} />
```
Comment (Bruception) [inline frontend/src/pages/Test/Test.jsx:27]: Try creating some test page components to see if you can switch pages.

Comment (Arturoo0) [reply frontend/src/pages/Test/Test.jsx:27]: You put the users

Comment (Arturoo0) [reply frontend/src/pages/Test/Test.jsx:27]: I never added that

**Pull request merged**